views/layout.hbs

```diff
@@ -1,5 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
+  <nav>
```

Comment:
You could use a hbs blocker for when the user is logged in only show the logout and otherwiese!